### ISSUE TITLE
Arxiv submission

### ISSUE THREAD

**Issue by choldgraf** [choldgraf/paper-pearc_2017#25]
Hey all - to submit to arxiv we need the latex files to execute all from the same directory (because I need to upload the files 1 by 1 and then it auto-builds the paper from what we give it).

I'm not a latex guru - so can somebody rearrange the file structure so that it will work with Arxiv's submission system? Maybe @arokem or @aculich ?

TODO
---
- [ ] Update the arxiv paper with the latest version when we submit

**Comment by arokem** [comment]
I'm on it.

**Comment by arokem** [comment]
https://github.com/arokem/pearc_paper_2017/tree/arxiv

**Comment by choldgraf** [comment]
do you want me to just pull from that to submit so we don't need to change this repo?

**Comment by arokem** [comment]
Yes - I don't think we should merge this mess into our pristine beautifully
organized master branch.

**Comment by choldgraf** [comment]
ok, submitted and I'll let you know when I hear something

**Comment by arokem** [comment]
We'll want to update arXiv with the last version when we're done. Maybe
keep this open, so we don't forget?

**Comment by choldgraf** [comment]
Good point
